.github/workflows/cwin.yml

```diff
@@ -0,0 +1,32 @@
+name: cwin
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      - name: Set git to use LF
+        run: |
+          git config --global core.autocrlf input
+
+      - uses: actions/checkout@v4
+
+      - name: Install Cygwin
+        uses: cygwin/cygwin-install-action@master
+        with:
+          packages: cmake gcc-fortran libjpeg-devel make ninja zlib-devel
+      - name: Test HDF4
+        run: |
+          export PATH=/usr/bin:$PATH
+          mkdir build
+          cd build
+          which cmake
+          cmake -G Ninja \
+           -D BUILD_SHARED_LIBS:BOOL=OFF \
+           -D CTEST_DROP_SITE_INIT:STRING="my.cdash.org" \
```

Comment:
If we're going to integrate CDash and GitHub CI, which I'm not sure we should do, we need to do that systematically and not ad-hoc.